cmd/admin-heal-ops.go

```diff
@@ -543,24 +541,59 @@ func (h *healSequence) healConfig() error {
 		return errServerNotInitialized
 	}
 
-	configFile := path.Join(minioConfigPrefix, minioConfigFile)
-	configBackupFile := path.Join(minioConfigPrefix, minioConfigBackupFile)
-	for _, cfg := range []string{configFile, configBackupFile} {
-		res, err := objectAPI.HealObject(h.ctx, minioMetaBucket, cfg, h.settings.DryRun)
+	marker := ""
```

Comment:
Can we also add a comment about the behaviour that server will try to check and heal the config always - irrespective of the bucket/object heal operation targeted.

Reply:
Yeah that's fine..